packages/app-store/_components/OmniInstallAppButton.tsx

```diff
@@ -21,11 +24,13 @@ export default function OmniInstallAppButton({
   className,
   returnTo,
   teamId,
+  eventType,
 }: {
   appId: string;
   className: string;
   returnTo?: string;
   teamId?: number;
+  eventType: EventTypeAppSettingsComponentProps["eventType"];
```

Comment:
I notice you're passing the whole event type object but we're only using the event type id. I think we should just pass the id down.